src/engine/client/serverbrowser.cpp

```diff
@@ -286,6 +286,7 @@ void CServerBrowser::Refresh(int RefreshFlags)
 		Packer.Reset();
 		Packer.AddRaw(SERVERBROWSE_GETINFO, sizeof(SERVERBROWSE_GETINFO));
 		Packer.AddInt(m_CurrentLanToken);
+		Packer.AddString("giex", 4);
```

Comment:
"giex" should also be a constant in mastersrv.h, using sizeof instead of 4.

Reply:
I did move to a constant, but I avoided sizeof because 4 is a property of the protocol rather then the constants size.